packages/kcmsf/src/pages/match.tsx

```diff
@@ -10,27 +10,90 @@ import { Judge } from "../utils/match/judge";
 import { expiryTimestamp, parseSeconds } from "../utils/time";
 
 type TimerState = "Initial" | "Started" | "Finished";
-
+type TeamFetchRes = {
```

Comment:
型名などのスコープが広い名称ではできるだけ略称を使わないほうが良いでしょう。
```suggestion
type TeamFetchResponse = {
```